Checkbox/Checkbox.module.less

```diff
@@ -4,32 +4,63 @@
 @import "../styles/skin.less";
 
 .checkbox {
+	display: inline-block;
+	position: relative;
+
 	.applySkins({
 		@size: @agate-checkbox-size;
 
 		width: @size;
 		height: @size;
-		border: @agate-checkbox-border-width solid @agate-checkbox-border-color;
 		border-radius: @agate-checkbox-border-radius;
 		color: @agate-checkbox-text-color;
 		background-color: @agate-checkbox-bg-color;
```

Comment:
Can we move this property to `.bg` since now it's a div for the background?

Reply:
Ok. I also moved the border-radius property to `.bg`.